parsons/zoom/zoom.py

```diff
@@ -153,3 +153,67 @@ def get_past_meeting_participants(self, meeting_id):
         tbl = self._get_request(f'report/meetings/{meeting_id}/participants', 'participants')
         logger.info(f'Retrieved {tbl.num_rows} participants.')
         return tbl
+
+    def get_meeting_registrants(self, meeting_id):
+        """
+        Get meeting registrants
```

Comment:
Add a period.*

* We could probably stand to have a style guide for documentation at some point.